pnpm-lock.yaml

```diff
@@ -9369,6 +9369,8 @@ importers:
       '@fluidframework/container-loader': '>=2.0.0-internal.3.4.0 <2.0.0-internal.4.0.0'
       '@fluidframework/eslint-config-fluid': ^2.0.0
       '@microsoft/api-extractor': ^7.34.4
+      '@fluidframework/protocol-definitions': ^1.1.0
+      '@microsoft/api-extractor': ^7.22.2
```

Comment:
Surprised to see this pop up. This is an older version than what we're using, and you didn't make any api-extractor-related changes. Might be worth re-running `pnpm i` from the root to (hopefully) clean this up.

Reply:
Removed.